Jordan Massingill

jordan-massingill opened the pull request:
Added picture of myself
Added meta tags to the <head> for bootstrap

jordan-massingill committed:
Initial Changes

Added picture of myself
Added meta tags to the <head> for bootstrap

groov1234 commented:
Hey, Jordan! Great usage of extra commit messages! Hope you're not struggling too much! `DM` me if you need assistance!!

jordan-massingill committed:
Content Alignment

Added Summary and Skills sections
Worked on trying to work my card through media queries
Added Jumbotron

jordan-massingill committed:
Modifications

Adjusted card to allow for better responsivity
Added a table to record past employment
Added background color to the middle content

jordan-massingill committed:
Additional Changes

Added work history
Added education
Experimented with background colors for tables and middle content
Still need to finish the card

jordan-massingill committed:
Edit for corrections

jordan-massingill committed:
Editing layout

jordan-massingill committed:
Late night modifications

Working on mobile layout with the card and the header

jordan-massingill committed:
Tinkering

Added a table to my card
Working with table responsivity
